tequilapi/client/dto.go

```diff
@@ -131,13 +131,15 @@ type SessionDTO struct {
 	Status          string `json:"status"`
 }
 
-// Services represents a list of running services on the node.
-type Services []Service
-
-// Service represents a service information.
-type Service struct {
-	ID       string          `json:"id"`
-	Status   string          `json:"status"`
-	Proposal ProposalDTO     `json:"proposal"`
-	Options  json.RawMessage `json:"options"`
+// ServiceListDTO represents a list of running services on the node
+type ServiceListDTO []ServiceInfoDTO
```

Comment:
I'm not sure if we need to name DTOs as `DTO` and arrays as `List`.
Is it a right golang-way?

Reply:
More than likely it's not: https://dave.cheney.net/practical-go/presentations/qcon-china.html#_dont_name_your_variables_for_their_types

Reply:
I am just keeping them aligned with Swagger definitions and being consistent with other endpoints.
Most likely we want to generate Tequilapi client from Swagger definitions later, instead of writing it handy